mods/beds/README.txt

```diff
@@ -5,7 +5,7 @@ See license.txt for license information.
 Authors of source code
 ----------------------
 Originally by BlockMen (MIT)
-Various Minetest developers and contributors (MIT)
+Various Luanti developers and contributors (MIT)
```

Comment:
strictly speaking they were contributors to "Minetest" at the time
or more specifically "Minetest Game"?